### PR TITLE
Update README file with the necessary external dependency

### LINKED ISSUE #161: Readme should be updated with the Jsyn External dependency.
**Actual Behaviour**
* Currently the readme file doesn't have a reference to JSyn external dependency that is used in the project for providing the audio-device interface.

**Expected Behaviour**
* The readme file should be updated accordingly.

**Would you like to work on the issue?**
I would like to work on this issue.

### PR DESCRIPTION
**Pull request by suresh-gandhi**
Fixes #161

**Changes**: 
* Updated the External dependencies section of the README file with JSyn External dependency.

**Checklist**: [Please tick following check boxes with `[x]` if the respective task is completed]
- [x] I have used resources from `strings.xml`, `dimens.xml` and `colors.xml` without hard-coding them
- [x] No modifications are done at the end of resource files `strings.xml`, `dimens.xml` or `colors.xml`
- [x] I have reformatted code in every file included in this PR [<kbd>CTRL</kbd>+<kbd>ALT</kbd>+<kbd>L</kbd>]
- [x] My code does not contain any extra lines or extra spaces
- [x] I have requested reviews from other members

### CODE REVIEW

**Comment by suresh-gandhi** [comment]
@iamareebjamal @CloudyPadmal @mariobehling Could you please review the PR.

**Review by adityastic — DISMISSED** [review]

**Comment by adityastic** [comment]
@suresh-gandhi Check Build Status, Its failing

**Comment by suresh-gandhi** [comment]
> @suresh-gandhi Check Build Status, Its failing

You mean Code Quality Review right? Let me see if I can do something about this. Thanks, man.

File: README.md [hunk 1]
```diff
@@ -17,4 +17,5 @@ Current devices in the medical industry are usually not accessible by doctors du
 A difference to existing projects like OpenBCI is that it will not be necessary to 3D print large headsets. Instead we are focusing on creating a device that collects as much data as possible through the forehead. To achieve this goal we are using high-grade sensors and flexible electronics.
 
 # External Dependencies
-*preference fix library* : com.takisoft.fix:preference-v7:$rootProject.prefFixLibraryVersion"
+* *preference fix library* : com.takisoft.fix:preference-v7:$rootProject.prefFixLibraryVersion"
+* *Library for providing the audio-device interface* : [JSyn Library for Android](http://www.softsynth.com/jsyn/beta/jsyn_on_android.php) | [github repo reference](https://github.com/iamareebjamal/jsyn).
```
Comment (iamareebjamal) [inline README.md]: No need to add all of these in README, we can see them in build.gradle

Comment (suresh-gandhi) [reply README.md]: Shall I provide the website link or leave that as well of the audio library.

**Comment by suresh-gandhi** [comment]
@CloudyPadmal @mariobehling resolved the conflicts. Could you please review and merge. Let me know if something else needs to be changed.

**Review by mariobehling — APPROVED** [review]

**Pull request merged**